modules/pytket-quantinuum/pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -359,6 +359,95 @@ def get_jobid(self, handle: ResultHandle) -> str:
         """
         return cast(str, handle[0])
 
+    def submit_qasm(
+        self,
+        qasm: str,
+        n_shots: int,
+        name: Optional[str] = None,
+        noisy_simulation: bool = True,
+        group: Optional[str] = None,
+        wasm_file_handler: Optional[WasmFileHandler] = None,
+        pytket_pass: Optional[BasePass] = None,
+        parametrized_zz: bool = False,
+        request_options: Optional[Dict[str, Any]] = None,
+    ) -> ResultHandle:
+        """Submit a qasm program directly to the backend.
+
+        :param qasm: QASM 2.0 program.
+        :type qasm: str
+        :param n_shots: Number of shots
+        :type n_shots: int
+        :param name: Job name, defaults to None
```

Comment:
This parameter is unused. (Should go in body of request?)

Reply:
I remember moving that line...must have got deleted somehow, this is the danger with big refactors I guess, nice catch!

Reply:
Have fixed and cleaned up a bit generally